packages/cli/test/integration/credentials/credentials.api.ee.test.ts

```diff
@@ -231,6 +247,22 @@ describe('GET /credentials', () => {
 // GET /credentials/:id - fetch a certain credential
 // ----------------------------------------
 describe('GET /credentials/:id', () => {
+	test('project viewers can view credentials', async () => {
+		const teamProject = await createTeamProject();
+		await linkUserToProject(member, teamProject, 'project:viewer');
+
+		const savedCredential = await saveCredential(randomCredentialPayload(), {
+			project: teamProject,
+		});
+
+		const response = await testServer
+			.authAgentFor(member)
+			.get(`/credentials/${savedCredential.id}`);
+
+		expect(response.statusCode).toBe(200);
+		expect(response.body.data).toBeDefined();
```

Comment:
I'm wondering whether we should return this or not, as the UI only shows the fields if the `credential:update` scope exists, so the fields won't be visible in the UI, and returning the data is just more potential to leakage wdyt? Maybe we can remove the `data` attribute

Reply:
Now I'm thinking that this might be the top-level `data` and not the actual credential information, so it should be fine. I'll approve in this case.

Reply:
I amended the assertions, it now asserts that the viewer cannot see the `data.data` attribute.